website/www/site/content/en/contribute/_index.md

```diff
@@ -21,55 +24,61 @@ See the License for the specific language governing permissions and
 limitations under the License.
 -->
 
-# Apache Beam Contribution Guide
+# Contribution guide
+
+<a class="arrow-list-header" data-toggle="collapse" href="#collapseOverview" role="button" aria-expanded="false"        aria-controls="collapseOverview">
+   {{< figure src="/images/arrow-icon_list.svg">}}
+
+## Overview
+
+</a>
+
+<div class="collapse dont-collapse-sm" id="collapseOverview">
 
-The Apache Beam community welcomes contributions from anyone!
+There are lots of opportunities to contribute. You can for example:
 
-If you have questions, please [reach out to the Beam community](/contribute/get-help).
+* ask or answer questions on [user@beam.apache.org](/community/contact-us/) or
+  [stackoverflow](https://stackoverflow.com/questions/tagged/apache-beam)
+* review proposed design ideas on [dev@beam.apache.org](/community/contact-us/)
+* file [bug reports](https://issues.apache.org/jira/projects/BEAM/issues)
+* review [changes](https://github.com/apache/beam/pulls)
+* work on the core programming model (what is a Beam pipeline and how does it
+  run?)
+* improve the developer experience (for example, Windows guides)
+* organize local meetups of users or contributors to Apache Beam
 
-There are lots of opportunities to contribute:
+...and many more. Most importantly, if you have an idea of how to contribute, then do it!
```

Comment:
This `...` doesn't render correctly: 
![image](https://user-images.githubusercontent.com/675055/106914840-430fef00-66ba-11eb-97c3-17224aaa7016.png)

Reply:
should be good now

Reply:
It still looks the same to me

Reply:
This is still not rendering correctly, I think we should just remove the "... and many more" unless you know what's going wrong here.

Reply:
hmm weird okay, I will try again, and if without success then I will simply remove it